### ISSUE TITLE
Checkout project and run tests: 47 tests. 2 passed, 45 failed

### ISSUE THREAD

**Issue by ascudder** [locks/ember-localstorage-adapter#119]
I wanted to try to help work on this project, so I checked it out with git and ran `bower install`. 

But, when I run the test command that's in the readme.md file, almost all of the tests are failing on me. Here's some sample output:
```
$ phantomjs test/runner.js test/index.html
WARNING: Please use `ember.debug.js` instead of `ember.js` for development and debugging.
DEPRECATION: A container should only be created for an already instantiated registry. For backward compatibility, an isolated registry will be instantiated just for this container.
        at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember/ember.js:1134
        at Container (file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember/ember.js:1142)
        at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/helpers.js:49
        at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/tests.js:50
        at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:176
        at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:358
        at process (file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:1453)
        at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:479
[...]
Test failed: DS.LSAdapter: find with id
    Failed assertion: Setup failed on find with id: Assertion Failed: You tried to set `adapter` property to an instance of `DS.Adapter`, where it should be a name
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember/ember.js:4840
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember-data/ember-data.js:9805
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember/ember.js:9957
    at get (file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember/ember.js:15176)
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/bower_components/ember/ember.js:29229
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/helpers.js:71
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/tests.js:50
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:176
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:358
    at process (file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:1453)
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:479
    Failed assertion: Died on test #2     at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/vendor/qunit.js:425
    at file:///Users/andrewscudder/Sites/ember-localstorage-adapter/test/tests.js:71: 'undefined' is not an object (evaluating 'store.find')
TypeError: 'undefined' is not an object (evaluating 'store.find')
[...]
Took 320ms to run 47 tests. 2 passed, 45 failed.
```

Am I doing something wrong? I was hoping to start with passing tests so that I could see when I made changes that might break something, but now I'm not sure how to proceed.

For reference, here are some versions of the software I'm running:
OS X: 10.10.3
Node: 0.12.4
npm: 2.11.1
Bower: 1.4.1
PhantomJS: 1.9.8

**Comment by kurko** [comment]
You're right, but I'm getting something different: `Assertion Failed: You tried to set `adapter` property to an instance of `DS.Adapter`, where it should be a name`.

I'm not really working with LS right now, so I don't much time to work on this. Would like to take a hold of it?

**Comment by ascudder** [comment]
Oh, yes, I'm seeing that error too. The output log was quite long so I just picked a couple of representative lines.

Anyway, I'll have to see if I can make enough time to work on a PR to get this working again. Thanks for confirming that it's not just an issue for me!

**Comment by locks** [comment]
I believe this is fixed by #130, I just tried master and everything passes :)

**Comment by ascudder** [comment]
Yup, works for me, too. Thanks!
